tools/grambase.tex

```diff
@@ -1,66 +0,0 @@
-\infannex{gram}{Grammar summary}
```

Comment:
No, why? It was only used as the starting point for `source/grammar.tex`, which now contains the contents from this file.